odoo/cli/__init__.py

```diff
@@ -1,3 +1,4 @@
 # Import just the command, the rest will get imported as needed
-from .command import Command, main  # noqa: F401
-from .help import Help  # noqa: F401
+
+from .command import Command, commands  # noqa: F401
```

Comment:
```suggestion
from .command import Command  # noqa: F401
```
why would we need to export `commands`? `help` can go and import it directly from `odoo.cli.command` doesn't it?